ceph/rbd/workflows/cleanup.py

```diff
@@ -0,0 +1,123 @@
+import json
+from time import sleep
+
+from ceph.rbd.utils import exec_cmd
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def pool_cleanup(client, pools, **kw):
+    """_summary_
+
+    Args:
+        rbd (_type_): _description_
+        pools (_type_): _description_
+    """
+    if kw.get("dir_name"):
+        exec_cmd(node=client, cmd="rm -rf {}".format(kw.get("dir_name")))
+
+    ceph_version = kw.get("ceph_version")
+    # mon_allow_pool_delete must be True for removing pool
+    if ceph_version and ceph_version >= 5:
+        exec_cmd(node=client, cmd="ceph config set mon mon_allow_pool_delete true")
+        sleep(20)
+
+    for pool in pools:
+        exec_cmd(
+            cmd=f"ceph osd pool delete {pool} {pool} " "--yes-i-really-really-mean-it",
+            node=client,
+        )
+
+
+def unmount(client, mount_point):
```

Comment:
Since the unmount operation is specific to cleaning up a created mount point in our case, I thought it was appropriate to retain it as part of cleanup workflow. Please let me know if and why you think otherwise.